### DISCUSSION TITLE
Releases calendar?

### DISCUSSION THREAD

**Discussion by thermseekr** [arendst/Tasmota#14532] [General]
Hello there,

Forgive me if this is something well known, I did a quick search and didn't find an answer. Is there a release calendar for new versions of Tasmota? Or, is there something that triggers a new release like a given number of merged PRs? I'm trying to guess when the next release will be available.

**Comment by barbudor** [comment]
With the advent of the CALVER (Calendar versioning), it is expected to get a "release" regularly every 2 month (it was more or less alerady the case before in fact). Next release should be 2022.02.
This is mainly to mark a step in time as the "release" version is generally nothing more than a "freeze" of the current development.
Maintenance and bug fix is always made on the development branch.
There is no roadmap or master plan to have a given feature or support a given device in release X Y or Z.

**Reply by sfromis** [reply]
The "releases calendar" back in time can be seen here: https://github.com/arendst/Tasmota/tags
